src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +18,34 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+    public static final double LOOP_PERIOD = 0.02; // [s]
+    public static final double NOMINAL_VOLTAGE = 12; // [volts]
+
+    //TODO: change to real values
+    public static final class Shooter {
+        public static final int TICKS_PER_ROTATION = 4096;
+
+        // NOTE: this is the only constants you need to change
+        // TODO: Calibrate
+        public static final double VELOCITY_TOLERANCE = 3; // [RPS]
+        public static final double MODEL_TOLERANCE = 3;
+        public static final double ENCODER_TOLERANCE = 0.1; // [ticks]
+        public static final double J = 0; //moment of inertia [kg * m^2]
+        public static final double ARBITRARY_FEED_FORWARD = 0;
```

Comment:
I took it from [DCMotor.java](https://github.com/wpilibsuite/allwpilib/blob/d6ed20c1e419284b353c2f13f67223cd43134f4f/wpimath/src/main/java/edu/wpi/first/wpilibj/system/plant/DCMotor.java#L179)

Reply:
That's cool, I didn't know it existed. But I didn't mean that.
You use [this](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/state-space/state-space-flywheel-walkthrough.html#modeling-using-flywheel-moment-of-inertia-and-gearing) method, but I think [this](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/state-space/state-space-flywheel-walkthrough.html#modeling-with-system-identification )method will be more accurate because it is based on empirical values of the exact motors you use.
IMO, using the moment and inertia gotten from CAD is less accurate than running frc-characterization. Also, I believe the constants from the file you provided were tested on one motor or a few, not on the specific motors you are using.

Reply:
I don't think that this discussion is really related to the purpose of this PR.
When we used the constants from `DCMotor.java`, it satisfied our needs, and if it won't satisfy out need this time, we can always change the constant when we calibrate our constants.
If you want, I can always add a `TODO` to this line...

Reply:
> it satisfied our needs

I see.
> If you want, I can always add a TODO to this line

Good idea.